addons/mail/static/tests/new/messaging_menu/messaging_menu_tests.js

```diff
@@ -418,3 +419,16 @@ QUnit.test("different mail.channel are not grouped", async function (assert) {
     await click(group_1);
     assert.containsOnce(target, ".o-mail-chat-window");
 });
+
+QUnit.test("mobile: active icon is highlighted", async function (assert) {
+    patchUiSize({ size: SIZES.SM });
+    await start();
+    await click(".o_menu_systray i[aria-label='Messages']");
+    await click(
+        document.querySelectorAll(".o-mail-messaging-menu-mobile-tab-icon")[1].closest("div")
+    );
+    assert.hasClass(
+        document.querySelectorAll(".o-mail-messaging-menu-mobile-tab-icon")[1].closest("div"),
+        "text-primary"
```

Comment:
I removed `.o-mail-messaging-menu-mobile-tab-icon` and added `..o-mail-messaging-menu-tab` on the div.
Also used `:contains(Chat)` instead of `[1]`.